sdk/src/types/block/output/mod.rs

```diff
@@ -252,6 +252,15 @@ impl Output {
         }
     }
 
+    /// Checks whether the output is an implicit account.
+    pub fn is_implicit_account(&self) -> bool {
+        if let Output::Basic(output) = self {
+            output.is_implicit_account()
+        } else {
+            false
+        }
```

Comment:
could be made into a one-liner I think:
```rust
matches!(self, Output::Basic(output) if output.is_implicit_account())
```

Reply:
Are you strongly opinionated about that? I think the if/let version is easier to humanly parse

Reply:
no, not all, just wanted to mention it if you'ld prefer it a bit shorter. I try to remember your preference.